src/connect.js

```diff
@@ -20,7 +20,10 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  const res = {};
+  return function(args) {
```

Comment:
you can write simply 
```suggestion
  return (args) => ({ ...params, ... args });
```